<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Viewport" name="viewport" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="parametersDockWidget">
   <property name="windowTitle">
    <string>Параметры</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="parametersDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="objectsListTab">
        <attribute name="title">
         <string>Сцена</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QScrollArea" name="objectsListTabScrollArea">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="objectsListTabScrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>278</width>
              <height>383</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="cameraLabel">
               <property name="text">
                <string>Камера</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cameraPositionLabel">
               <property name="text">
                <string>Положение:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="cameraPositionLayout">
               <item>
                <widget class="QDoubleSpinBox" name="cameraPositionX">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="prefix">
                  <string>X: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="cameraPositionY">
                 <property name="prefix">
                  <string>Y: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="cameraPositionZ">
                 <property name="prefix">
                  <string>Z: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="cameraLookAtLabel">
               <property name="text">
                <string>Точка взгляда:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="cameraLookAtLayout">
               <item>
                <widget class="QDoubleSpinBox" name="cameraLookAtX">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="prefix">
                  <string>X: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="cameraLookAtY">
                 <property name="prefix">
                  <string>Y: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="cameraLookAtZ">
                 <property name="prefix">
                  <string>Z: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="objectsLabel">
               <property name="text">
                <string>Список объектов:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="objectsListWidget">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="selectionRectVisible">
                <bool>true</bool>
               </property>
               <property name="currentRow">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="objectLabel">
               <property name="text">
                <string>Выбранный объект:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="objectPositionLabel">
               <property name="text">
                <string>Положение:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="objectPositionLayout">
               <item>
                <widget class="QDoubleSpinBox" name="objectPositionX">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="prefix">
                  <string>X: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectPositionY">
                 <property name="prefix">
                  <string>Y: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectPositionZ">
                 <property name="prefix">
                  <string>Z: </string>
                 </property>
                 <property name="minimum">
                  <double>-1000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="objectRotationLabel">
               <property name="text">
                <string>Поворот:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="objectRotationLayout">
               <item>
                <widget class="QDoubleSpinBox" name="objectRotationX">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="prefix">
                  <string>X: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectRotationY">
                 <property name="prefix">
                  <string>Y: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectRotationZ">
                 <property name="prefix">
                  <string>Z: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="objectScaleLabel">
               <property name="text">
                <string>Масштаб:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="objectScaleLayout">
               <item>
                <widget class="QDoubleSpinBox" name="objectScaleX">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="prefix">
                  <string>X: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectScaleY">
                 <property name="prefix">
                  <string>Y: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="objectScaleZ">
                 <property name="prefix">
                  <string>Z: </string>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="createObjectsTab">
        <attribute name="title">
         <string>Создать</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QScrollArea" name="createTabScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="createTabScrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-272</y>
              <width>261</width>
              <height>583</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="widthLabel">
               <property name="text">
                <string>Толщина фотоаппарата</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="width">
               <property name="value">
                <double>40.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lengthLabel">
               <property name="text">
                <string>Длина фотоаппарата</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="length">
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>140.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="heightLabel">
               <property name="text">
                <string>Высота фотоаппарата</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="height">
               <property name="value">
                <double>70.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="radiusLabel">
               <property name="text">
                <string>Радиус объектива</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="radius">
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lensWidthLabel">
               <property name="text">
                <string>Ширина объектива</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lensWidth">
               <property name="value">
                <double>30.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lensMountLengthLabel">
               <property name="text">
                <string>Длина крепления объектива</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lensMountLength">
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lensMountWidthLabel">
               <property name="text">
                <string>Ширина крепления объектива</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lensMountWidth">
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="marginWidthLabel">
               <property name="text">
                <string>Ширина свободной части</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="marginWidth">
               <property name="value">
                <double>40.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sideButtonsHeightLabel">
               <property name="text">
                <string>Высота боковых кнопок</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sideButtonsHeight">
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="shutterButtonHeightLabel">
               <property name="text">
                <string>Высота кнопки спуска затвора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="shutterButtonHeight">
               <property name="value">
                <double>15.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sideButtonsRadiusLabel">
               <property name="text">
                <string>Радиус боковых кнопок</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sideButtonsRadius">
               <property name="value">
                <double>15.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="shutterButtonRadiusLabel">
               <property name="text">
                <string>Радиус кнопки спуска затвора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="shutterButtonRadius">
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="createChangeButtonsLayout">
               <item>
                <widget class="QPushButton" name="createModelButton">
                 <property name="text">
                  <string>Создать</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="editModelButton">
                 <property name="text">
                  <string>Изменить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>О Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewport</class>
   <extends>QWidget</extends>
   <header>Widgets/Viewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
